### PR TITLE
Dashboard api

### PR DESCRIPTION
**Pull request by scrummish**

### CODE REVIEW

File: components/people/person-card.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { useTheme } from "@emotion/react";
 
 export const PersonCard = ({ person, showTitle = false, anchorName }) => {
   const theme = useTheme();
-
+  console.log(person)
```
Comment (Woozl) [inline components/people/person-card.js]: remove

File: components/projectSpotlight.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { useTheme } from '@mui/material/styles'
 import { MarkdownLess } from './markdown'
 
 export const ProjectCard = ({project}) => {
+  console.log(project)
```
Comment (Woozl) [inline components/projectSpotlight.js]: remove

File: utils/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    headers: {
+      accept: "application/json",
+    },
+  };
+  const mergedOptions = {
+    ...defaultOptions,
+    ...options,
+  };
+  const requestUrl = getDashboardURL(endpoint);
+  process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0;
```
Comment (Woozl) [inline utils/dashboard.js]: What was the reason for this? Is this only happening in dev? Node docs highly discourage its use 

From: https://nodejs.org/api/cli.html
> If value equals '0', certificate validation is disabled for TLS connections. This makes TLS, and HTTPS by extension, insecure. The use of this environment variable is strongly discouraged.

File: components/projects/search-bar.js
Comment (Woozl) [inline components/projects/search-bar.js:1]: Wasn't this in a different branch? Is this supposed to be here again?

**Review by Woozl — COMMENTED** [review]
I think this is a pretty good start at this. There are a couple small notes I made on the code and some questions for clarifications. There are 3 places I noticed are missing or broken that may be fixable on our end or simply limitations of the dashboard API that we'll need Sandy to add:

1. Project spotlight missing cover images: <details><summary>screenshot</summary> 
   image</details>
2. Project/Collaboration/Teams page people lists are missing images: <details><summary>screenshot</summary>    image</details>
3. Project page has descriptions that are no longer truncated. Also this will probably require an API change but the the projects page now takes ~12 seconds to load/hydrate because the people endpoint returns a 10k line/345kb json response. <details><summary>screenshot</summary>    image</details>

**Comment by Woozl** [comment]
Two things I forgot to mention. We discussed yesterday the "our work" menu has collaborations/researchGroups/teams that it's currently still pulling from strapi. Maybe we should migrate this to the dashboard data in this PR as well, if it isn't too much work. 

Also, @mbwatson when you do your review, don't forget to be on the VPN, thought I'd mention it since it wasn't and tripped me up for a bit.

**Comment by mbwatson** [comment]
> Also, @mbwatson when you do your review, don't forget to be on the VPN, thought I'd mention it since it wasn't and tripped me up for a bit.

@Woozl thanks for this reminder!

**Review by mbwatson — COMMENTED** [review]
i see all the same things David noticed (thanks for your thorough inspection!)

we'll want to figure out which we can address and which would be better addressed with an API change.

File: utils/dashboard.js (hunk 3 above)
Comment (Woozl) [reply utils/dashboard.js]: Dashboard API is over http currently, we will remove this once that has a certificate.

**Comment by mbwatson** [comment]
## for us to change, client-side

- remove some debug logging
- truncate project descriptions
- on indiv proj pages & people page, use `api/webinfo/people/{id}/photo` endpoint URLs instead of base64 images
- on project list page and indiv proj page, use featuredImage URL
- remove TLS env var when SSL is complete

## API changes to request

- prefer image URLs to base64-encoding, can remove images from big people list
- `/divisions`, or similar, to get Our Work menu lists

File: utils/dashboard.js (hunk 3 above)
Comment (Woozl) [reply utils/dashboard.js]: The dashboard is now using SSL at `https://dashboard.renci.org` so we can remove this

**Review by Woozl — APPROVED** [review]

**Pull request merged**